functions/data_test/data_test.py

```diff
@@ -1,33 +1,35 @@
 from profiling import profile_data
 from suite_run import validate_data
 import os
-import s3fs
-import re
 import boto3
 import awswrangler as wr
 import json
-from datasource import prepare_final_ds
-from datasource import get_source_name
-from datasource import get_file_extension
+from datasource import prepare_final_ds, get_source_name, get_file_extension
 
 
 def handler(event, context):
-    s3 = boto3.resource("s3", endpoint_url=f"http://{os.environ['S3_HOST']}:4566") if os.environ['ENVIRONMENT'] == 'local' else boto3.resource("s3")
+    endpoint_url = f"http://{os.environ['S3_HOST']}:4566"
```

Comment:
let's move out hardcoded port number to variable

Reply:
done